### DISCUSSION TITLE
Accidently transferred storage to non-existent Personal account

### DISCUSSION THREAD

**Discussion by paramaggarwal** [vercel/community#5556] [Account Issue] [answered]
### Question

I accidently transferred a Vercel Postgres storage to "Personal" account from the modal popup from Storage settings, but I do not have any way to access this Personal account from the Vercel dashboard.

Please help, I have lost access to my database!

Team ID: `team_2flpk70sLnDsaUAufs6uTOEI`
Vercel ID: `kAbWRLCGCo9gMiLiFY7rsHcz`

**Comment by swarnava** [comment]
Hello Param, let us check this and get back to you!

**Reply by paramaggarwal** [reply]
Let me know if further information is required.

**Comment by paramaggarwal** [comment]
Please help me get back access to this database - it still exists as the POSTGRES_URL still works, but I cannot find it anywhere in my Vercel account. It has got transferred to "Personal" account which does not show in my Vercel Dashboard.

**Comment by swarnava** [comment, marked as answer]
Hi Param, we have made some changes in your acco. Can you login to your personal account and transfer storage back to team so we can revert it?

Let us know once it’s done

**Comment by swarnava** [comment]
Hi @paramaggarwal, please revert to us once this is done so we can revert your account to original state.

**Comment by paramaggarwal** [comment]
Thanks a lot @swarnava - I was able to transfer back the storage from the personal to team account. My issue is resolved. 🎉 

Request you to kindly **not revert** the account to original state as anyways the personal account will get converted to a personal team in the upcoming weeks as per this update: https://vercel.com/changelog/2024-01-account-changes

**Comment by paramaggarwal** [comment]
One more thing @swarnava, it is not exactly clear what limit I've crossed here: ([link to this store](https://vercel.com/cbx1/~/stores/postgres/store_wTeifAxqUXRBDG7r/data))

<img width="1247" alt="Screenshot 2024-01-25 at 6 57 00 AM" src="https://github.com/vercel/community/assets/543981/372bf118-fb1d-4522-b654-fcc071f00f68">
